src/Microsoft.Azure.ServiceBus/Primitives/TaskExtensionHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus.Primitives
+{
+    using System;
+    using System.Threading.Tasks;
+
+    public class TaskExtensionHelper
+    {
+        public static void Schedule(Func<Task> func)
+        {
+            Task.Run(async () =>
+            {
+                try
+                {
+                    await func();
+                }
+                catch (Exception)
+                {
```

Comment:
If you look at how the inner tasks that are passed to this function are written, all of them are self contained, in the sense that they try catch everything that happen in the function internally.  So I did not see a real need for logging here again. But in any case, it might be a good idea to log, I'll add a todo to log any unexpected exception here.